CruGlobal/godtools-swift
# GT-2266 default tool language

Hey @levieggertcru, it took me a while to wrap my head around this, especially because I kept swapping language ID for language code 😅 so let me know if I'm missing anything here.

To test:
Set app language to anything besides Vietnamese, go into tool details for "The Four" and favorite the Vietnamese version.  When you tap back out onto the favorites page, you'll see the tool in Vietnamese and be able to open it in Vietnamese.
